src/core/Microsoft.PowerToys.Settings.UI.UnitTests/ViewModelTests/ImageResizer.cs

```diff
@@ -155,7 +155,7 @@ public void EncoderShouldUpdateValueWhenSuccessful()
 
             // Assert
             viewModel = new ImageResizerViewModel(mockSettingsUtils, SettingsRepository<GeneralSettings>.GetInstance(mockGeneralSettingsUtils.Object), SendMockIPCConfigMSG);
-            Assert.AreEqual("163bcc30-e2e9-4f0b-961d-a3e9fdb788a3", viewModel.GetEncoderGuid(viewModel.Encoder));
+            Assert.AreEqual("163bcc30-e2e9-4f0b-961d-a3e9fdb788a3", ImageResizerViewModel.GetEncoderGuid(viewModel.Encoder));
```

Comment:
this line feels awkward.  Feels like we should be able to get the GUID from just an extension overload on encoder.

Reply:
it looks like the Encoder property here is just an integer value representing the index of the encoder in the image resizer dropdown, and the method fetches the appropriate guid based on this index

Reply:
feels weird how to invoke this.  not your code.  Maybe create a new issue to get that adjusted